README.md

```diff
@@ -1 +1,2 @@
-# best-repo-ever
\ No newline at end of file
+# best-repo-ever
+This is some content. :)
```

Comment:
Good work adding text to a file!
You're mastering the staging / commit / push-pull process :)